kotsadm/Makefile

```diff
@@ -34,6 +34,8 @@ build-release:
 	docker push kotsadm/kotsadm:${GIT_TAG}
 	mkdir -p bin/docker-archive/kotsadm
 	skopeo copy docker-daemon:kotsadm/kotsadm:${GIT_TAG} docker-archive:bin/docker-archive/kotsadm/${GIT_TAG}
+	mkdir -p bin/docker-archive/dex
+	skopeo copy docker://quay.io/dexidp/dex:v2.26.0 docker-archive:bin/docker-archive/dex/${GIT_TAG}
```

Comment:
~${GIT_TAG}~ v2.26.0?

Reply:
i just copied what was done with postgres https://github.com/replicatedhq/kots/blob/master/kotsadm/migrations/Makefile#L11-L14

Reply:
hmm, that looks wrong... but ok I guess. don't want to break anything